packages/quantic/cypress/integration/recent-results-list/recent-results-list-expectations.ts

```diff
@@ -0,0 +1,97 @@
+import {InterceptAliases} from '../../page-objects/search';
+import {should} from '../common-selectors';
+import {
+  RecentResultsListSelectors,
+  RecentResultsListSelector,
+} from './recent-results-list-selectors';
+
+const urlResult = 'https://github.com/coveo/ui-kit/';
+
+function recentResultsListExpectations(selector: RecentResultsListSelector) {
+  return {
+    displayPlaceholder: (display: boolean) => {
+      selector
+        .placeholder()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the placeholder`);
+    },
+    displayLabel: (display: boolean) => {
+      selector
+        .label()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the label`);
+    },
+    displayResults: (display: boolean) => {
+      selector
+        .results()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display results`);
+    },
+    displayResult: (value: string, display: boolean) => {
+      selector
+        .result(value)
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(
+          `${should(display)} display recent result with value: "${value}"`
+        );
+    },
+    displayEmptyList: (display: boolean) => {
+      selector
+        .emptyList()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display empty list`);
+    },
+    numberOfResults: (length: number) => {
+      selector
+        .results()
+        .should('have.length', length)
+        .logDetail(`should display ${length} results in the list`);
+    },
+    lastResultContains: (result: string) => {
+      selector
+        .lastResult()
+        .contains(result)
+        .logDetail(`The last result added should contain "${result}"`);
+    },
+    labelContains: (label: string) => {
+      selector
+        .label()
+        .contains(label)
+        .logDetail(`The label should contain "${label}"`);
+    },
+    hrefContainsAtResult: (value: string) => {
+      selector
+        .result(value)
+        .should('have.attr', 'href')
+        .and('include', urlResult)
+        .then((href) => expect(href).to.equal(urlResult))
```

Comment:
These two lines seem to validate the same thing.

The `.then((href) => expect...)` seems to be stricter, so can we keep only this one and remove the `.and('include', urlResult)`?